lib/index.js

```diff
@@ -310,7 +331,7 @@ function clsCheck (msg) {
 
 exports.clsCheck = clsCheck
 
-if (!serviceKey) {
+if (!cleansedKey) {
```

Comment:
Would it make sense to move this closer to line 268

Reply:
It make sense to do something with it. It can issue a redundant message now, e.g., bad trace-mode, can't load bindings, etc. so the disabling might not be true (it could already be disabled). i'll have to think about what to do because it should either move up (and other potential messages check it) or stay where it is but modify the message and/or logic to be consistent.